src/Nethermind/Nethermind.Specs/ChainSpecStyle/ChainSpecLoader.cs

```diff
@@ -414,7 +417,7 @@ private static void LoadGenesis(ChainSpecJson chainSpecJson, ChainSpec chainSpec
         if (withdrawalsEnabled)
             genesisHeader.WithdrawalsRoot = Keccak.EmptyTreeHash;
 
-        if(depositsEnabled)
+        if (depositsEnabled)
```

Comment:
we have the same check two times, depositsEnabled and isEIP6110Enabled, please cleanup

Cosmetic reaname isEip7002Enabled here to existsEnabled